.travis.yml

```diff
@@ -11,20 +11,20 @@ before_install:
 
 install:
 
-script: "make -j VERBOSE=1 NINJA=1 $TARGET"
+script: "python tools/run-tests.py $OPTS"
 
```

Comment:
Not sure whether it makes much of a difference given our small project size but it would be great if Travis CI could still use Ninja for the build :)